compiler/rustc_middle/src/ty/query.rs

```diff
@@ -311,6 +343,18 @@ macro_rules! define_callbacks {
 
             fn try_mark_green(&'tcx self, tcx: TyCtxt<'tcx>, dep_node: &dep_graph::DepNode) -> bool;
 
+            /// SAFETY: :ferrisClueless:
+            unsafe fn execute(
```

Comment:
nils wtf lmao

Reply:
i'm not sure whether a trait object with over 300 methods is less cursed tbh